doc/adr/0005-discovery-documents.md

```diff
@@ -0,0 +1,34 @@
+# 5. Use Discovery Documents for requirements and technical choices
+
+Date: 2019-10-11
+
+## Status
+
+Accepted
+
+## [Context](https://github.com/libero/community/issues/23)
+
+Libero product features and architectural choices benefit from the standardization of *discovery documents* that enable collaboration by being shared between team members, across teams, and outside of eLife.
+
+## Decisions
+
+- Always track discovery work in at least one dedicated Github issue.
+- Prefix Google Docs discovery documents titles with `Discovery document:`
+- Move them into the single [`Libero` shared drive](https://drive.google.com/drive/u/1/folders/0AC6f8fXdeXmMUk9PVA).
+- Link them from the relevant Github issue.
+- Establish a timebox in calendar time at the start of the discovery activity.
+- Organize in questions and answers; some of the answers are allowed to be assumption if clearly stated that they are.
+- Link to a [Github milestone](https://github.com/libero/publisher/milestone/20) if applicable.
+- Add a `TL;DR` section at the top before closing the document at the end of the timebox.
+
+## Consequences
```

Comment:
What do you feel about adding a section here about the expect artefacts of the discovery process, for example you would expect potentially a set of options and recommendations as output? Or a backlog of work?

Reply:
Can add a very obvious consequence, the presence of a document; not sure if the rest is consistent across various discoveries; for example, requirements discovery of use cases vs. technical discovery on tools and technologies